src/backend/main/py.pi/caliopen_pi/features/mail.py

```diff
@@ -189,7 +189,14 @@ def _compute_pi(self, participants, features):
                             'version': 0})
 
     def process(self, message, participants):
-        """Process the message for privacy features and PI compute."""
+        """
+        Process the message for privacy features and PI compute.
+
+        :param message: a message parameter that will be updated with PI
+        :ptype message: NewMessage
```

Comment:
I'll fix in another branch this documentation.I didn't see appearing this NewInboundMessage parameter